lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/examples/timer/TimerMock.java

```diff
@@ -45,9 +45,9 @@
     this.seconds = seconds;
   }
 
-  public static void main(final String[] args) throws BindException, InjectionException, Exception {
+  public static void main(final String[] args) throws BindException, InjectionException, InterruptedException {
     final Configuration c = TimerMock.CONF
-        .set(TimerMockConf.MOCK_SLEEP_TIME, 1)
+        .set(TimerMockConf.MOCK_SLEEP_TIME, 5)
```

Comment:
Why do we need to change this? And we probably need to keep this code in sync with code in `README.md`